### DISCUSSION TITLE
Welcome to TOPPERS discussion!

### DISCUSSION THREAD

**Discussion by hiroakitakada** [toppers/users-forum#1] [General]
TOPPERSに関するQ&Aや情報交換の場です。

**Comment by ryoon** [comment]
小野寺と申します。
このorganizationのdiscussionsで議論を追加できないのですが、こちらには記載できるようなので、どなたか気付いていただければと思います。
New discussionといったボタンが表示されるはずと思っているのですが、見当たらないようです。

また、https://github.com/toppers/users-forum からリンクされている https://github.com/toppers/users/discussions/new を開いても、 You can't perform that action at this time.  を表示されて、新しい議論を追加できません。

何かユーザーで設定すべき事項があり、設定できていないでしょうか?
過去には同じ環境から新しい議論を追加した経験もあり、何が変わったのかと考えています。

**Reply by mitsut** [reply]
@ryoon 
小野寺様、ご連絡ありがとうございます。

関連しそうな設定を見直しましたので、New discussionが表示して投稿できるかご確認いただけませんでしょうか

**Reply by ryoon** [reply]
@mitsut 早速ご対応いただきありがとうございます。New discussionのボタンが表示されるようになり、新規投稿もできました。

**Reply by mitsut** [reply]
ご確認ありがとうございました。引き続きよろしくお願いいたします。

**Reply by hiroakitakada** [reply]
さらに，README.mdの「New discussion」からのリンク先を更新しました。
